### PR TITLE
[DOC UPDATE] Adds t-digest content

### PR DESCRIPTION
**Pull request by nermiller**
Updates the index page with t-digest description, tips, and reference

### CODE REVIEW

File: docs/docs/_index.md [hunk 1]
```diff
@@ -8,19 +8,87 @@ type: docs
...
-RedisBloom adds four probabilistic data structures to Redis: a scalable **Bloom filter**, a **cuckoo filter**, a **count-min sketch**, and a **top-k**. These data structures trade perfect accuracy for extreme memory efficiency, so they're especially useful for big data and streaming applications.
+RedisBloom adds these probabilistic data structures to Redis: a scalable **Bloom filter**, a **cuckoo filter**, a **count-min sketch**, a **top-k**, and . These data structures trade perfect accuracy for extreme memory efficiency, so they're especially useful for big data and streaming applications.
 
 **Bloom and cuckoo filters** are used to determine, with a high degree of certainty, whether an element is a member of a set.
 
 A **count-min sketch** is generally used to determine the frequency of events in a stream. You can query the count-min sketch get an estimate of the frequency of any given event.
 
 A **top-k** maintains a list of _k_ most frequently seen items.
 
+**t-digest** estimates quantiles based on a data stream or a large dataset of floating-point values.
```
Comment (nermiller) [inline docs/docs/_index.md:19]:
```suggestion
**t-digest** estimates percentiles based on a data stream or a large dataset of floating-point values.
```

File: docs/docs/_index.md [hunk 2]
```diff
@@ -8,19 +8,87 @@ type: docs
 [![Discord](https://img.shields.io/discord/697882427875393627?style=flat-square)](https://discord.gg/wXhwjCQ)
 [![GitHub](https://img.shields.io/static/v1?label=&message=repository&color=5961FF&logo=github)](https://github.com/RedisBloom/RedisBloom/)
 
-RedisBloom adds four probabilistic data structures to Redis: a scalable **Bloom filter**, a **cuckoo filter**, a **count-min sketch**, and a **top-k**. These data structures trade perfect accuracy for extreme memory efficiency, so they're especially useful for big data and streaming applications.
+RedisBloom adds these probabilistic data structures to Redis: a scalable **Bloom filter**, a **cuckoo filter**, a **count-min sketch**, a **top-k**, and . These data structures trade perfect accuracy for extreme memory efficiency, so they're especially useful for big data and streaming applications.
```
Comment (LiorKogan) [inline docs/docs/_index.md:11]: missing: t-digest

Comment (nermiller) [inline docs/docs/_index.md:11]:
```suggestion
RedisBloom adds these probabilistic data structures to Redis: a scalable **Bloom filter**, a **cuckoo filter**, a **count-min sketch**, a **top-k**, and **t-digest**. These data structures trade perfect accuracy for extreme memory efficiency, so they're especially useful for big data and streaming applications.
```

File: docs/docs/_index.md [hunk 3]
```diff
@@ -8,13 +8,17 @@ type: docs
...
-A **count-min sketch** is generally used to determine the frequency of events in a stream. You can query the count-min sketch get an estimate of the frequency of any given event.
+* **HyperLogLog**: How many unique values appeared so far in the data stream?
+* **Bloom filter** and **Cuckoo filter**: Did the value v already appear in the data stream?
+* **Count-min sketch**: How many times did the value v appear in the data stream?
+* **Top-K**: What are the k most frequent values in the data stream?
 
-A **top-k** maintains a list of _k_ most frequently seen items.
+Answering each of these questions accurately can require a huge amount of memory, but if you are willing to sacrifice accuracy, you can reduce the memory requirements drastically. Each of these data structures allows you to set a controllable tradeoff between accuracy and memory consumption.
 
 **t-digest** estimates percentiles based on a data stream or a large dataset of floating-point values.
```
Comment (LiorKogan) [inline docs/docs/_index.md:23]: Please remove this line, and add t-digest to the list above, with the questions it answers as sub-bullets.

**Review by LiorKogan — APPROVED** [review]

**Pull request merged**